src/main/java/core/basesyntax/NumberBall.java

```diff
@@ -0,0 +1,11 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class NumberBall {
+    public int getRandomNumber() {
+        Random randomNumber = new Random();
+        int value = randomNumber.nextInt(100);
+        return value;
+    }
```

Comment:
There are redundant empty lines after the class declaration and method signature. Please remove them to adhere to the coding style guidelines.

Reply:
deleted the extra line